java/ql/src/semmle/code/java/PrintAst.qll

```diff
@@ -97,7 +97,7 @@ private string getQlClass(Top el) {
 private predicate locationSortKeys(Element ast, string file, int line, int column) {
   exists(Location loc |
     loc = ast.getLocation() and
-    file = loc.getFile().toString() and
+    file = loc.getFile().getAbsolutePath() and
```

Comment:
This file is shared across languages as far as I know:
https://github.com/github/codeql/blob/main/cpp/ql/src/semmle/code/cpp/PrintAST.qll

Reply:
Ah right, thanks for pointing that out. But it looks like it contains language specific code and is therefore not part of [`identical-files.json`](https://github.com/github/codeql/blob/main/config/identical-files.json), so maybe this change is fine then?

Reply:
Indeed, you're right.